### DISCUSSION TITLE
Unable to Add Project to Existing Team

### DISCUSSION THREAD

**Discussion by 0x3639** [community/community#31678] [Other Feature Feedback, Questions, & Ideas]
I'm trying to add an Existing Project called ZIP Framework (https://github.com/orgs/Zenon-Community/projects/1) to the team BICHz under the Zenon-Network Organization.  The 2 team members are Admins.

When I try to add the Project to the BICHz team "ZIP Framework" is not available.   
https://github.com/orgs/Zenon-Community/teams/bichz/projects

How can I add an existing Project to an Existing Team?

<img width="453" alt="Screen Shot 2022-09-01 at 8 17 09 PM" src="https://user-images.githubusercontent.com/5406130/188038485-21c3524e-af72-48a2-ae36-4d9275691fc5.png">

**Comment by jlunder00** [comment]
This is apparently on the roadmap: https://github.com/github/roadmap/issues/563

The index only searches for classic projects, and not the new projects, but you can't create classic projects unless your organization already had them before the new projects went live. This is rather frustrating, as I am having the same issue, and this functionality was essentially removed when projects went live and classic projects began being phased out, since for new organizations, the page is completely useless until this issue is resolved and shipped. We can only hope they'll ship this soon.
